lib/models/sponsor.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:ecellapp/core/res/strings.dart';
+
+/// Model to get All Info about Sponsor it's Capsuled by SponsorData
+class Sponsor {
+  //int data
```

Comment:
coordinate with backend team, others in app team and refer design, and remove unimportant fields that are not important w.r.t. the app.